src/components/Icon.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const COLORS = ['#6D6B6D','#DCD9D8','#D33833','#335061','#81B0C4','#709aaa','#000'];
+
+function Icon({title}) {
+    title = title
+        .replace('Jenkins ', '')
```

Comment:
is this duplicated from that helper? not ideal

Reply:
Yea, and not even the same logic, gross, fixing